e2e/tests/components/menu/menu.e2e-spec.ts

```diff
@@ -9,6 +9,14 @@ describe('Menu', () => {
     beforeEach(async () => {
         page = new MenuPage();
         await page.getPage();
+
+        // set the browser window to a specific size to ensure consistency
+        await browser.driver.manage().window().setSize(1320, 800);
```

Comment:
Could this be reverted? It's hard to tell if there were any visual regressions when all the screenshots have changed. In general try to avoid making changes that impact screenshots unless the screenshot shows behavior that we intentionally corrected.